src/e2e-test/features/bigquery/source/BigQueryToGCS.feature

```diff
@@ -124,33 +124,33 @@ Scenario:Validate successful records transfer from BigQuery to GCS with bucket h
   When Select plugin: "BigQuery" from the plugins list as: "Source"
   When Expand Plugin group in the LHS plugins list: "Sink"
   When Select plugin: "GCS" from the plugins list as: "Sink"
-  Then Open BigQuery source properties
-  Then Enter BigQuery property reference name
-  Then Enter BigQuery property projectId "projectId"
-  Then Enter BigQuery property datasetProjectId "projectId"
+  Then Navigate to the properties page of plugin: "BigQuery"
+  And Enter input plugin property: "referenceName" with value: "Reference"
+  And Replace input plugin property: "project" with value: "projectId"
+  And Enter input plugin property: "datasetProject" with value: "datasetprojectId"
+  And Replace input plugin property: "dataset" with value: "dataset"
   Then Override Service account details if set in environment variables
-  Then Enter BigQuery property dataset "dataset"
-  Then Enter BigQuery source property table name
-  Then Validate output schema with expectedSchema "bqSourceSchemaDatatype"
+  And Enter input plugin property: "table" with value: "bqSourceTable"
+  Then Click on the Get Schema button
   Then Validate "BigQuery" plugin properties
   Then Close the BigQuery properties
-  Then Open GCS sink properties
-  Then Enter the GCS sink mandatory properties
-  Then Close the GCS properties
+  Then Navigate to the properties page of plugin: "GCS"
+  Then Enter input plugin property: "referenceName" with value: "sourceRef"
```

Comment:
Add the step to enter project id

Reply:
Changes Added as per the review comment.